test/test/kernel/KernelTests.java

```diff
@@ -38,7 +38,7 @@ public void fdtransfer(TestProcess p) throws Exception {
     @Test(mainClass = ListFiles.class, jvmArgs = "-XX:+UseParallelGC -Xmx1g -Xms1g", os = {Os.MACOS, Os.WINDOWS})
     public void notLinux(TestProcess p) throws Exception {
         try {
-            p.profile("-e cpu -d 3 -i 1ms -o collapsed -f %f --fdtransfer", true);
+            p.profile("-e cpu -d 3 -i 1ms -o collapsed -f %f --fdtransfer", false);
```

Comment:
The argument may be removed altogether: sudo=false is the default.

Reply:
done